tools/cloud-build/daily-tests/tests/hcls-v6.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+test_name: hcls-cluster-v6
+deployment_name: "hcls-v6-{{ build }}"
+# No non-alphanumerical characters in the slurm cluster name - they will be
+# removed by HPC Toolkit slurm wrappers, which will break the playbook
+slurm_cluster_name: "hclsv6{{ build[0:4] }}"
+zone: us-central1-c
+workspace: /workspace
+blueprint_yaml: "{{ workspace }}/examples/hcls-blueprint-v6.yaml"
+max_nodes: 5
```

Comment:
Is this used at all? I think it is old artifact.

Reply:
Removed.